types.cc

```diff
@@ -2013,6 +1976,12 @@ struct compare_visitor {
 }
 
 int32_t abstract_type::compare(bytes_view v1, bytes_view v2) const {
+    if (v1.empty()) {
+        return v2.empty() ? 0 : -1;
+    }
+    if (v2.empty()) {
+        return 1;
+    }
     return visit(*this, compare_visitor{v1, v2});
```

Comment:
This slows down types that don't have special empty values, like text and blob.

Reply:
I think this commit should just be dropped.

Reply:
dropped it.